.travis.yml

```
@@ -7,7 +7,6 @@ cache:
 before_script:
   npm install
 script:
-  npm test
   npm run build
```

Please add hyphes for all the things so that we're less likely to hit the issue again.
```suggestion
  - npm run build
```
same for the `before_script`. Notice how travis shows all there recommended configs with hypenated lists for all the things, even lists of 1: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#ember-apps